Makefile

```diff
@@ -21,7 +21,10 @@ vet:
 
 # Run tests
 test: fmt vet
-	KUBEBUILDER_ASSETS="$(shell $(GOBIN)/setup-envtest use -p path 1.19.x)" go test -v  ./api/... ./pkg/controllers/... ./pkg/...  -coverprofile cover.out
+	KUBEBUILDER_ASSETS="$(shell $(GOBIN)/setup-envtest use -p path 1.26.x)" go test -v  ./api/... ./pkg/controllers/... ./pkg/...  -coverprofile cover.out
+
+e2e: ;$(info $(M)...Run e2e test.) @ ## Run e2e test in kind.
```

Comment:
We should integrate `e2e` into GitHub workflows.

Reply:
```
  e2e:
    name: e2e test
    needs: build
    runs-on: ubuntu-22.04
    steps:
      - name: checkout code
        uses: actions/checkout@v3
        with:
          # Number of commits to fetch. 0 indicates all history for all branches and tags.
          # We need to guess version via git tags.
          fetch-depth: 0
      - name: install Go
        uses: actions/setup-go@v3
        with:
          go-version: 1.19
      - name: setup e2e test environment and run e2e
        run: |
          hack/kind_e2e.sh


```

Reply:
Thanks for the tip, I will add it to GitHub workflows later.